src/components/paymentTable/paymentTable.scss

```diff
@@ -0,0 +1,14 @@
+.gray-text {
+  color: #3a3a3a;
+}
+.table thead th {
+  vertical-align: bottom;
+  border-bottom: 2px solid #2f80ed;
+}
+
+.table td,
+.table th {
+  padding: 0.75rem;
+  vertical-align: top;
+  border-top: 1px solid #2f80ed;
+}
```

Comment:
I think you can nest them like this try it
.table { 
thead th {
  vertical-align: bottom;
  border-bottom: 2px solid #2f80ed;
}
 td,
 th {
  padding: 0.75rem;
  vertical-align: top;
  border-top: 1px solid #2f80ed;
}
}